en: Add operator version requirement for telemetry config (#3095)
pingcap/docs: telemetry.md

See [Deploy TiDB Operator in Kubernetes](https://docs.pingcap.com/tidb-in-kubernetes/stable/deploy-tidb-operator) for details.

> **Note:**
>

> > This configration item requires at least TiDB Operator v1.1.2 to take effect.

Reviewer: ```suggestion
> This configuration item requires TiDB Operator v1.1.2 or later to take effect.
```

See [Deploy TiDB Operator in Kubernetes](https://docs.pingcap.com/tidb-in-kubernetes/stable/deploy-tidb-operator) for details.

> **Note:**
>

> > This configration item requires at least TiDB Operator v1.1.2 to take effect.

Reviewer: ```suggestion
> This configuration item requires TiDB Operator v1.1.2 or later to take effect.
```
